src/Sulu/Bundle/ActivityBundle/Domain/Model/Activity.php

```diff
@@ -26,12 +26,12 @@ class Activity implements ActivityInterface
     private $type;
 
     /**
-     * @var mixed[]
+     * @var mixed
```

Comment:
i think `mixed[]` is the correct type here - why did you change it? 🤔

Reply:
Since the `phpstan/phpstan-doctrine` `0.12.40` release, the error in the pull request description above occurrs for json fields ... And it's true, the database "could" contain also a `bool` in that column

Reply:
Would add it as ignored error and not add casting to array here.

Reply:
This kind of things is okay to be a runtime error, the checks are unnecessary costs in this cases.